assets/images/exploration_statistics_icons/unres_ans.svg

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Nope -- please put this in assets/images/icons/unresolved_answers.svg . The current dirname is too specific.